guide/src/error_handling.md

```diff
@@ -94,28 +102,52 @@ and then only in errors that correspond to a missing secret:
 {{#include ../samples/src/error_handling.rs:update-secret-not-found}}
 ```
 
-If this is a "not found" error, we try to create the secret. This will simply
-return on failures:
+If this is a "not found" error, you can try to create the secret. This simply
+returns on failure:
 
 ```rust,ignore
 {{#include ../samples/src/error_handling.rs:update-secret-create}}
 ```
 
-Assuming the creation of the secret is successful, we can try to create the
-secret version again, this time just returning an error if anything fails:
+Assuming [`create_secret`](#create_secret) is successful, you can
+try to add the secret version again, this time just returning an error if
+anything fails:
 
 ```rust,ignore
 {{#include ../samples/src/error_handling.rs:update-secret-try-again}}
 ```
 
+## What's next
+
+Learn more about error handling:
+
+- [Examine error details]
+- [Handling binding errors]
+
 ______________________________________________________________________
 
-## Error handling: complete code
+## Code samples
+
+For the complete sample, see [error_handling.rs].
+
+### `update_attempt`
 
 ```rust,ignore
-{{#include ../samples/src/error_handling.rs:update-secret}}
+{{#include ../samples/src/error_handling.rs:update-attempt}}
+```
+
+### `create_secret`
+
+```rust,ignore
+{{#include ../samples/src/error_handling.rs:create-secret}}
 ```
 
 [configuring retry policies]: /configuring_retry_policies.md
+[error_handling.rs]: https://github.com/pcoet/google-cloud-rust/blob/main/guide/samples/src/error_handling.rs
```

Comment:
We definitely should not link to `pcoet`. Linking to `main` (vs. a specific commit) can also be problematic if we rename the file.

We already dump the contents above, I don't think we need to link to the sample code too.

Reply:
Oops, didn't mean to use my project! Removed entirely.